From 42f4728f6d90ed48dbd0680130fd9e88a3703d80 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Mon, 10 Jun 2019 12:03:23 -0700
Subject: [PATCH] joyent/node-gerritbot#6 Pass "-S none" to SSH commands
 joyent/node-gerritbot#7 Update "javascriptlint" and "jsstyle" submodules
 Reviewed by: Alex Wilson <alex.wilson@joyent.com> Approved by: Alex Wilson
 <alex.wilson@joyent.com>

---
 .gitmodules           | 4 ++--
 deps/javascriptlint   | 2 +-
 deps/jsstyle          | 2 +-
 lib/event-stream.js   | 1 +
 lib/ssh-cmd-runner.js | 1 +
 package.json          | 2 +-
 6 files changed, 7 insertions(+), 5 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index f80629a..f9dad4c 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,6 +1,6 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = https://github.com/davepacheco/javascriptlint
+	url = https://github.com/joyent/javascriptlint
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle
+	url = https://github.com/joyent/jsstyle
diff --git a/deps/javascriptlint b/deps/javascriptlint
index 040bf5e..ad52812 160000
--- a/deps/javascriptlint
+++ b/deps/javascriptlint
@@ -1 +1 @@
-Subproject commit 040bf5e429969ae42d976b570fd64c9e17fab20f
+Subproject commit ad52812e77bdfb1e90fb71a1201adb2b665a27e6
diff --git a/deps/jsstyle b/deps/jsstyle
index b94eb62..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit b94eb629ee524d21b563c8b4461caa2bdf1becc6
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
diff --git a/lib/event-stream.js b/lib/event-stream.js
index 167e362..768409d 100644
--- a/lib/event-stream.js
+++ b/lib/event-stream.js
@@ -100,6 +100,7 @@ GerritEventStream.prototype.state_spawning = function (S) {
 
 	this.gw_kid = mod_cproc.spawn('ssh',
 	    ['-v', '-p', this.gw_port,
+	    '-S', 'none',
 	    '-o', 'ServerAliveInterval=10',
 	    '-o', 'ServerAliveCountMax=1',
 	    this.gw_user + '@' + this.gw_host,
diff --git a/lib/ssh-cmd-runner.js b/lib/ssh-cmd-runner.js
index 8c0fada..32acf87 100644
--- a/lib/ssh-cmd-runner.js
+++ b/lib/ssh-cmd-runner.js
@@ -120,6 +120,7 @@ SSHCmdRunner.prototype.state_spawning = function (S) {
 	this.scr_kid = mod_cproc.spawn('ssh',
 	    ['-v', '-p', this.scr_port,
 	    '-E', this.scr_fifo,
+	    '-S', 'none',
 	    '-o', 'ServerAliveInterval=10',
 	    '-o', 'ServerAliveCountMax=1',
 	    this.scr_user + '@' + this.scr_host,
diff --git a/package.json b/package.json
index 85e8b29..133293b 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "gerritbot",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "description": "",
   "main": "lib/index.js",
   "scripts": {
-- 
2.21.0

